src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not be empty after the class declaration. Remove the redundant empty line at the beginning of the class body.